File: atcoder/abc192/abc192_b.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main{
    
    public static void main(String[] args) {
        BufferedReader bf = new BufferedReader(new InputStreamReader(System.in));
        try {
            String line = bf.readLine();
            for(int i = 0; i < line.length(); i++){
                if((i+1)%2 == 0 && !Character.isUpperCase(line.charAt(i))){
                    System.out.println("No");
                    return;
                }
                else if((i+1)%2 != 0 && Character.isUpperCase(line.charAt(i))){
                    System.out.println("No");
                    return;
                }
            }
            System.out.println("Yes");
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: atcoder/abc192/abc192_b.kt
package atcoder

import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    val In = BufferedReader(InputStreamReader(System.`in`))
    val s = In.readLine().toCharArray()
    var difficult = true
    var i = 0
    while(i < s.size && difficult) {
        if(i%2 == 1) {
            if (s[i] > 'Z')
                difficult = false
        } else {
            if (s[i] < 'a')
                difficult = false
        }
        i += 1
    }
    if(difficult)
        println("Yes")
    else
        println("No")
}